src/components/AgreementSections/LinesList.js

```diff
@@ -57,7 +59,18 @@ export default class LinesList extends React.Component {
     type: line => <EResourceType resource={getResourceFromEntitlement(line)} />,
     count: line => <EResourceCount resource={getResourceFromEntitlement(line)} />,
     coverage: line => <CoverageStatements statements={line.coverage} />,
-    isCustomCoverage: line => (line.customCoverage ? <CustomCoverageIcon /> : ''),
+    isCustomCoverage: line => {
+      if (!line.customCoverage) return '';
+      return (
+        <Tooltip
+          text={<FormattedMessage id="ui-agreements.customcoverages.tooltip" />}
```

Comment:
We prefer to alphasort our props in ERM modules. Ie, `id` before `text`. Just makes it easier to find props when dealing with components that have a bunch of them.